homeassistant/components/axis/strings.json

```
@@ -13,13 +13,13 @@
       }
     },
     "error": {
-      "already_configured": "Device is already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "device_unavailable": "Device is not available",
```

There is also a common string for a connection error.
```suggestion
      "device_unavailable": "[%key:common::config_flow::error::cannot_connect%]",
```
or
```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
```

---

Agreed, done.